packages/SwingSet/test/bundles/.gitignore

```diff
@@ -1 +1,2 @@
 disk-bundle
+!.
```

Comment:
I don't believe that works. The exclusion needs to be collocated with the ignore pattern

Reply:
I bet you're thinking of [this in the gitignore docs](https://git-scm.com/docs/gitignore):
> It is not possible to re-include a file if a parent directory of that file is excluded

In this case, it's the directory itself that's being re-included.

I can confirm it works on my machine. Please let me know if it doesn't on yours.

Reply:
I verified and it doesn't work on my machine with the command I used above

Reply:
```
node ➜ /workspaces/agoric/agoric-sdk (ta/gitignore_agstate) $ find . -not -path './.git/*' -not -path './packages/xsnap/moddable/*' -not -path './packages/xsnap/xsnap-native/*' | git check-ignore --stdin -v --no-index
.gitignore:54:bundles   ./packages/SwingSet/test/bundles
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/vat-named.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/bootstrap-bundles.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/vat-disk.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/.gitignore
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/test-bundles-controller.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/test-bundles-kernel.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/vat-install.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/test-bundles.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/nonvat-importable.js
.gitignore:54:bundles   ./packages/SwingSet/test/bundles/nonvat-install.js
node ➜ /workspaces/agoric/agoric-sdk (ta/gitignore_agstate) $ cat ./packages/SwingSet/test/bundles/.gitignore 
disk-bundle
!.
```

Reply:
```
node ➜ /workspaces/agoric/agoric-sdk (ta/gitignore_agstate) $ touch ./packages/SwingSet/test/bundles/foo.js
node ➜ /workspaces/agoric/agoric-sdk (ta/gitignore_agstate) $ git status
On branch ta/gitignore_agstate
Your branch is behind 'origin/ta/gitignore_agstate' by 3 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)

nothing to commit, working tree clean
```

Reply:
Ah, I was testing existing files this way:
- `git ls-files packages/SwingSet/test/` showed the files
- edited a file and git status showed it dirty

But I didn't test adding a new file to the directory,
- `touch packages/SwingSet/test/bundles/newfile` doesn't find anything

So I see you're right that the exclusion was a no-op. I'll PR a fix.